common/src/main/scala/com/gu/stripe/StripeService.scala

```diff
@@ -15,9 +15,18 @@ class StripeService(config: StripeConfig, client: FutureHttpClient, baseUrl: Str
   val wsUrl = baseUrl
   val httpClient: FutureHttpClient = client
 
-  override def wsPreExecute(req: Request.Builder): Request.Builder =
+  override def wsPreExecute(req: Request.Builder): Request.Builder = {
     req.addHeader("Authorization", s"Bearer ${config.secretKey}")
 
+    config.version match {
```

Comment:
Since this is side-effect-y, I would use `foreach` instead of `match`.

```
req.addHeader("Authorization", s"Bearer ${config.secretKey}")

config.version foreach { version =>
  logger.info(s"Making a stripe call with version: $version")
  req.addHeader("Stripe-Version", version)
}

req
```

Also, there is no need for `{}` to separate `case` statements

Reply:
Ah, good point. I shall use foreach instead.